components/brave_sync/jslib_messages.h

```diff
@@ -109,11 +110,11 @@ class SyncRecord {
 
   enum Action {
     A_INVALID = -1,
-    CREATE = 0,
-    UPDATE = 1,
-    DELETE = 2,
-    A_MIN = CREATE,
-    A_MAX = DELETE,
+    A_CREATE = 0,
```

Comment:
this seems really hacky, but I think we'll replace this with the chromium sync enums later anyway

Reply:
chromium doesn't use raw DELETE either
```
  enum SyncChangeType {
    ACTION_INVALID,
    ACTION_ADD,
    ACTION_UPDATE,
    ACTION_DELETE,
  };
```